packages/flutter/lib/src/widgets/scrollable.dart

```diff
@@ -430,6 +453,13 @@ class ScrollableState extends State<Scrollable> with TickerProviderStateMixin, R
   @override
   void didChangeDependencies() {
     _updatePosition();
+    if (widget.isDefaultScrollAction) {
+      final _DefaultScrollActionState? defaultScrollAction = DefaultScrollAction.maybeOf(context);
+      // Similar to notificationContext, _ScrollableScope is placed above the
+      // widget using it: RawGestureDetector, which allows us to access this
+      // ScrollableState
+      defaultScrollAction?._defaultScrollKey = _gestureDetectorKey;
```

Comment:
What if we have two scrollables with `isDefaultScrollAction` set to true?

Reply:
In some situations that would probably be illegal. In others (e.g. if they are in separate routes) it would presumably be ok?